show-me-the-visualization/app.html

```diff
@@ -46,17 +46,34 @@
         "encoding": { "x": { "field": "x", "type": "quantitative" } },
         "layer": [
           { "mark": { "type": "rule", "strokeDash": [4, 8] } },
-          { "mark": { "type": "text", "align": "left", "dx": 4, "text": "Root" } }
+          { "mark": { "type": "text", "align": "left", "dx": 4, "text": "root" } }
         ]
       };
       const iterations_scatter = {
         "data": {
           "values": iterations
         },
         "encoding": {
-          "x": { "field": "x", "type": "quantitative" },
-          "y": { "field": "y", "type": "quantitative" },
-          "text": { "field": "index", "type": "nominal" }
+          "x": {
+            "field": "x",
+            "type": "quantitative",
+            "axis": {
```

Comment:
This styling is not shown in the README on line 404-420. Please add there as well

Reply:
Updated part of `iterations_scatter`